test/testuninitvar.cpp

```diff
@@ -4002,7 +4002,18 @@ class TestUninitVar : public TestFixture {
                         "        someType_t gVar;\n"
                         "        if(gVar.flags[1] == 42){}\n"
                         "}\n");
-        ASSERT_EQUALS("[test.cpp:7]: (error) Uninitialized variable: flags\n", errout.str());
+        ASSERT_EQUALS("[test.cpp:7]: (error) Uninitialized variable: gVar\n", errout.str());
```

Comment:
hmm.. I think it would be more "proper" to warn about gVar.flags here. I want to warn about "pcdata" in the test case below because the pointer is uninitialized. What do you say? Should we track uninitialized struct members for uninitialized pointers at all?

Reply:
I updated the messages.